terraform/testcases/otlp_metric_amp_mock/otconfig.tpl

```diff
@@ -0,0 +1,30 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
+
+processors:
+  batch:
+
+exporters:
+  logging:
+    loglevel: info
```

Comment:
Most of the other tests set the logging level to debug.

Reply:
The other tests do not run the log exporter. Look at the exports portion of the test

Reply:
I can change this to what other tests do if you think I should

Reply:
https://github.com/aws-observability/aws-otel-test-framework/blob/a61d96e03129b20f252ab41b256de6faf5ffa540/terraform/testcases/otlp_metric/otconfig.tpl#L13-L24

Reply:
It's more of a nitpick. All of the other tests use debug.